GliaWidgets/ViewController/GliaViewController.swift

```diff
@@ -48,7 +50,7 @@ class GliaViewController: UIViewController {
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
-
```

Comment:
Hey, found another trailing whitespace 😄 

I think it would be very beneficial if you tried to build _all_ of the branches we created on Monday and look through the SwiftLint warnings, now that you have it installed. You can then run `swiftlint --fix` if needed.